src/sonic-yang-models/yang-templates/sonic-acl.yang.j2

```diff
@@ -152,7 +152,7 @@ module sonic-acl {
 
 				leaf ETHER_TYPE {
 					type string {
-					    pattern "0x0[6-9a-fA-F][0-9a-fA-F]{2}|0x[1-9a-fA-F][0-9a-fA-F]{3}";
+                                            pattern "0x0[6-9a-fA-F][0-9a-fA-F]{2}|0x[1-9a-fA-F][0-9a-fA-F]{3}|153[6-9]|15[4-9][0-9]|1[6-9][0-9][0-9]|[2-9][0-9]{3}|[1-5][0-9]{4}|6553[0-5]|655[0-2][0-9]|65[0-4][0-9]{2}|6[0-4][0-9]{3}";
```

Comment:
>                                             [](http://example.com/codeflow?start=0&length=44)

Mixing tab and spaces in the same file. Could you make them consistent?

Reply:
The entire file is not properly intended. I can perform indentation but will show as multiple changes. Is that fine?

Reply:
Aligned it as before